# Frozen ortholog-transfer records with receiver-training-only mapping


Fixed pilot: existing Plasmodium schizont spatial-prediction labels projected through native orthogroups onto the same 3,827 eligible Toxoplasma compartment genes and seed-17 nested split as the first kNN pilot. The projection context has no receiver target column. Receiver mapping fits only on training labels. Stage/context equivalence and independent biological truth remain unresolved; no data is added or replaced.

In [1]:
from scripts.freeze_transfer_pilot import freeze
summary=freeze('results/ortholog_transfer_pilot_2026_10_08')
summary

{'strategy': 'ortholog_transfer', 'receiver': 'Tg:compartment:label_calls:v1', 'donor': 'Pf:lopit_pf_location:label_calls:v1', 'receiver_grade': 'prediction', 'donor_grade': 'prediction', 'benchmark_admitted': False, 'eligible_candidate_genes': 3827, 'retained_test_rows': 560, 'mapped_receiver_genes': 967, 'mapped_training_genes': 550, 'mapped_test_genes': 126, 'counts': {'eligible': 560, 'answered': 123, 'abstained': 437, 'correct': 65, 'wrong': 58, 'unique_biological_entities': 560}, 'metrics': {'accuracy': 0.11607142857142858, 'coverage': 0.21964285714285714, 'precision_of_calls': 0.5284552845528455, 'macro_precision': 0.2732323232323232, 'macro_recall': 0.16611567277328146, 'macro_f1': 0.18536839030353014, 'weighted_f1': 0.1584609403558611, 'kappa': 0.09771792943841731, 'mcc': 0.16428134834044153, 'macro_auroc': None, 'macro_auprc': None}, 'baseline_metrics': {'accuracy': 0.23035714285714284, 'coverage': 1.0, 'precision_of_calls': 0.23035714285714284, 'macro_precision': 0.009598214

All 560 test genes are retained, including missing projections and unsupported donor categories. Native transfer provides no confidence or class-score matrix. Counts, per-class outcomes and baselines use the identical receiver test cohort. Numerical source values, installed strategies and calibration are unchanged. Full outer coverage and the other missing label-capable adapters remain open under 64.10.